editor/plugins/org.fusesource.ide.wsdl2rest/src/org/jboss/fuse/wsdl2rest/impl/ResourceMapperImpl.java

```diff
@@ -0,0 +1,128 @@
+package org.jboss.fuse.wsdl2rest.impl;
```

Comment:
missing header

Reply:
We will be pulling this code in jarred form from maven, so I won't change it at this time, but a good spot.